Actually disable the fontations code flag since we don't build that library. Otherwise chromium crashes with a null dereference

--- src/third_party/blink/renderer/platform/runtime_enabled_features.json5.orig	2025-04-21 18:29:28.190307211 +0200
+++ src/third_party/blink/renderer/platform/runtime_enabled_features.json5	2025-04-21 23:33:49.257769003 +0200
@@ -2165,11 +2165,9 @@
     },
     {
       name: "FontationsFontBackend",
-      status: "stable",
     },
     {
       name: "FontationsForSelectedFormats",
-      status: "stable",
     },
     {
       name: "FontFamilyPostscriptMatchingCTMigration",
